test/tests/functional/pbs_dup_acc_log_for_resv.py

```diff
@@ -51,6 +52,52 @@ def setUp(self):
             self.skipTest('test requires two MoMs as input, ' +
                           'use -p moms=<mom1>:<mom2>')
 
+    def check_log_resv_time(self, accounting_log_msg, start_tup,
+                            end_tup, check_status):
+        """
+        This is a helper function to reterive start and end time
+        of standing rervation from accounting log.
+        """
+        find_start = m[start_tup][end_tup].find("start=")
```

Comment:
Hi  @bhroam as per discussion with you , I have incorporated this comment. 
Done.